### PR TITLE
Database

### PR DESCRIPTION
**Pull request by Shrewstar**
#13

### CODE REVIEW

File: CreateTables.sql [hunk 1]
```diff
@@ -0,0 +1,90 @@
+CREATE DATABASE COP4331;
+USE COP4331;
+CREATE TABLE User (
+ID int NOT NULL AUTO_INCREMENT,
+firstName VARCHAR(50) NOT NULL DEFAULT '',
+lastName VARCHAR(50) NOT NULL DEFAULT '',
+username VARCHAR(50) NOT NULL DEFAULT '',
+password VARCHAR(50) NOT NULL DEFAULT '',
+dateCreated DATE NOT NULL DEFAULT CURRENT_TIMESTAMP,
+PRIMARY KEY (ID),
```
Comment (nashirj) [inline CreateTables.sql:10]: Please add indentation here to make it easier to read.

File: CreateTables.sql [hunk 2]
```diff
@@ -0,0 +1,90 @@
+CREATE DATABASE COP4331;
```
Comment (nashirj) [inline CreateTables.sql:1]: `CREATE DATABASE IF NOT EXISTS COP4331;`

File: CreateTables.sql [hunk 3]
```diff
@@ -0,0 +1,90 @@
+CREATE DATABASE COP4331;
+USE COP4331;
+CREATE TABLE User (
```
Comment (nashirj) [inline CreateTables.sql:3]: `CREATE TABLE IF NOT EXISTS User`

File: CreateTables.sql [hunk 4]
```diff
@@ -0,0 +1,90 @@
+CREATE DATABASE COP4331;
+USE COP4331;
```
Comment (nashirj) [inline CreateTables.sql:2]: Newline here please

File: CreateTables.sql [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+CREATE TABLE User (
+ID int NOT NULL AUTO_INCREMENT,
+firstName VARCHAR(50) NOT NULL DEFAULT '',
+lastName VARCHAR(50) NOT NULL DEFAULT '',
+username VARCHAR(50) NOT NULL DEFAULT '',
+password VARCHAR(50) NOT NULL DEFAULT '',
+dateCreated DATE NOT NULL DEFAULT CURRENT_TIMESTAMP,
+PRIMARY KEY (ID),
+) ENGINE = InnoDB;
+CREATE TABLE Contacts (
```
Comment (nashirj) [inline CreateTables.sql:12]: Newline here please

Comment (nashirj) [inline CreateTables.sql:12]: `CREATE TABLE IF NOT EXISTS Contacts`

File: CreateTables.sql [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+ID int NOT NULL AUTO_INCREMENT,
+firstName VARCHAR(50) NOT NULL DEFAULT '',
+lastName VARCHAR(50) NOT NULL DEFAULT '',
+email VARCHAR(100) NOT NULL DEFAULT '',
+phone VARCHAR(50) NOT NULL DEFAULT '',
+userID INT,
+PRIMARY KEY (ID),
+FOREIGN KEY (UserID) REFERENCES Users(ID)
```
Comment (nashirj) [inline CreateTables.sql:20]: Please indent

File: CreateTables.sql [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+) ENGINE = InnoDB;
```
Comment (nashirj) [inline CreateTables.sql:21]: Newline here please

**Review by nashirj — CHANGES_REQUESTED** [review]
Overall looks great, ty. Can you also add something to the README about how to use this script to initialize the database and about how to clear the values if you want to reset the database state? Thank you for doing this.

**Review by mplekunov — APPROVED** [review]
Good Job! LGTM

**Pull request merged**